airbyte-cdk/python/airbyte_cdk/utils/stream_status_utils.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+from datetime import datetime
+
+from airbyte_cdk.models import AirbyteMessage, AirbyteStreamStatusTraceMessage, AirbyteTraceMessage, StreamDescriptor, TraceType
+from airbyte_cdk.models import Type as MessageType
+
+
+def as_airbyte_message(stream, current_status, successful) -> AirbyteMessage:
```

Comment:
nit: it would be useful to add type annotations
`def as_airbyte_message(stream: Stream, current_status: AirbyteStreamStatus, successful: Optional[bool]) -> AirbyteMessage`

Reply:
Done.